.github/workflows/docker-publish-lakefs-rclone-export.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
         with:
           context: deployments/tools/export
           push: true
+          platforms: linux/amd64,linux/arm64,darwin/amd64,darwin/arm64
```

Comment:
Do we need to sign the Darwin images, or does Docker get a free pass on MacOS?

Reply:
docker images play different game - depending on the repository you publish to. so nothing extra we do at this point.